src/leo-client-app/src/pages/schedule-commands/[satId].tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+import { withPageAuthRequired } from "@auth0/nextjs-auth0/client";
+import Navbar from "@/components/navbar/Navbar";
+import Scheduler from "@/components/Scheduler";
+import { Stack } from "@mui/material";
+import { useRouter } from "next/router";
+
+function SchedulerPage() {
+  const router = useRouter();
+  let { satId } = router.query as {
+    satId: string;
+  };
```

Comment:
seems like this cna be removed ?

Reply:
think its good to leave in case we wanted to add more stuff to it